app/src/main/java/com/pileproject/drive/programming/visual/block/selection/IfThereWasALargeSoundBlock.java

```diff
@@ -42,8 +42,6 @@ public IfThereWasALargeSoundBlock(Context context) {
 
     @Override
     protected boolean evaluateCondition(MachineController controller) {
-        // need multiply 10 because getdB returns 10 times value
-        // the comment is messed up
-        return ((NxtController) controller).getSoundSensorValue() > mThreshold * 10;
```

Comment:
fixed in [here](https://github.com/PileProject/drivecommand/blob/develop/src/main/java/com/pileproject/drivecommand/model/nxt/NxtProtocol.java#L111)